### ISSUE TITLE
Farm run reminder

### ISSUE THREAD

**Issue by Khajgold** [runelite/runelite#8333]
**Is your feature request related to a problem? Please describe.**
Missed farm runs due to lack of attention to the widget

**Describe the solution you'd like**
A clear OSD reminding to do farm runs
Ability to choose its location/size
Ability to choose which patches you are interested in
Ability to group patches (or do by default) to remind when ALL have grown

**Describe alternatives you've considered**
Tray notification instead of OSD (if selected should override default setting)

**Additional context**
Basically a very clear and simple reminder to do your farm run or herb run.

**Comment by Abextm** [comment]
Duplicate #2212
